app/code/Magento/Cms/Block/Adminhtml/Wysiwyg/Images/Tree.php

```diff
@@ -115,7 +115,7 @@ public function getTreeLoaderUrl()
 
         $currentTreePath = $this->getRequest()->getParam('current_tree_path');
 
-        if (strlen($currentTreePath)) {
+        if ($currentTreePath && strlen($currentTreePath)) {
```

Comment:
Hello @anzin 
I think we need to check `$currentTreePath` on null, because '0' in condition is false.

```suggestion
        if ($currentTreePath !== null && strlen($currentTreePath)) {
```